baseband/vlbi_base/base.py

```diff
@@ -107,10 +119,46 @@ def _get_time(self, header):
         return header.time
 
     @lazyproperty
-    def time0(self):
-        """Start time."""
+    def start_time(self):
+        """Start time of file."""
         return self._get_time(self.header0)
 
+    @deprecated('0.X', name='time0', alternative='start_time',
+                obj_type='attribute')
+    def get_time0(self):
+        return self.start_time
+
+    time0 = property(get_time0, None, None)
+
+    @property
+    def current_time(self):
```

Comment:
Thinking about this more, how about just calling this `time`? I think it is unambiguous what it is, shorter, and easier to find. Also in the docstring, perhaps refer to `start_time` and `stop_time`.